doc/jsonrpc.md

```diff
@@ -9973,6 +9974,44 @@ Example response:
 }
 ~~~
 
+### bdev_lvol_shallow_copy {#rpc_bdev_lvol_shallow_copy}
```

Comment:
Should we need `bdev_lvol_shallow_copy_status`? The response contains `status` and `progress`.